Readme.md

```diff
@@ -3,15 +3,40 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E5000)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How to implement standalone and in-Form cascading SelectBoxes
-<!-- run online -->
-**[[Run Online]](https://codecentral.devexpress.com/e5000/)**
-<!-- run online end -->
 
-This example demonstrates how to implement cascading select boxes in the following scenarios:
-- Standalone SelectBoxes
+# SelectBoxes for DevExtreme - How to implement standalone and in-Form cascading SelectBoxes
+
+This example demonstrates how to implement cascading SelectBoxes in the following scenarios:
+
+- Ыtandalone SelectBoxes
 - SelectBoxes in Form
 
-The main idea is to get a changed value from the first editor and filter the data source in the second editor so that it shows relevant items.
+When you select a value from the first SelectBox, the second SelectBoxloads the filtered values. In this example, when you select a state only the cities from this state appear in the second SelectBox.
```

Comment:
`the second SelectBoxloads the filtered values` Do you mean 'the second SelectBox loads the filtered values'?

```suggestion
When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only the cities from the selected state appear in the second SelectBox.
```